entries/r/ramp.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Ramp": {
+    "domain": "ramp.com",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support.ramp.com/hc/en-us/articles/5775209997715",
+    "keywords": [
+      "payments",
```

Comment:
The other entries in the `payments` category are actual payment platforms or services, so this one does not really fit in there, I guess.
```suggestion
```